soccer/Processor.cpp

```diff
@@ -483,6 +449,7 @@ void Processor::run() {
 
         // Send motion commands to the robots
         sendRadioData();
+        _manual_control_node->run();
```

Comment:
Could you add a comment describing that the order of these two functions is very specific? If sendRadioData is called after, it will override all manual control if the robot is not visible since it sets the `intent.is_active`

Reply:
I've changed it so that `intent.is_active` is not touched if it's being controlled by a joystick